source/wiki/css-units.md

```diff
@@ -8,4 +8,50 @@ version: 2.1
 
 <!-- toc -->
 
-Work in progresss.
+## CSS `color` units
+
+```css
+.classA {
+  /* 3-chars hex */
+  color: #0f0;
+  /* 6-chars hex */
+  color: #00ff00;
+  /* rgba */
+  color: rgb(255, 0, 0);
+  /* rgba */
+  color: rgba(255, 0, 0, 0.5);
+  /* transparent */
+  color: transparent;
+  /* Basic color keywords */
+  color: orange;
+  /* Extended color keywords */
+  color: darkgray;
+}
+```
+
+## Notes
+
+* Not support `hsl()`, `hsla()`, `currentColor`, or 8-character hexadecimal color.
+
+* Performance of `rgb(a,b,c)` or `rgba(a,b,c,d)` is much worse than other color formats, please select the appropriate color format.
+
+build-in color name you can see [Color Names](./color-names.html).
+
+## CSS `length` units
+
+In weex we only support `px` length units., and it will resolve to a numeric type in the JavaScript runtime and native renderer.
```

Comment:
. and , shouldn't be together. We can rephrase it to following - 
In weex we only support `px` length units. It will resolve to a numeric type in the JavaScript runtime and native renderer.

Reply:
Thank you for your review, I will fix it.